binding/Binding/SkiaApi.generated.cs

```diff
@@ -12369,42 +12369,42 @@ private partial class Delegates {
 		// sk_fontstyleset_t* sk_fontmgr_match_family(sk_fontmgr_t*, const char* familyName)
 		#if !USE_DELEGATES
 		[DllImport (SKIA, CallingConvention = CallingConvention.Cdecl)]
-		internal static extern sk_fontstyleset_t sk_fontmgr_match_family (sk_fontmgr_t param0, [MarshalAs (UnmanagedType.LPStr)] String familyName);
+		internal static extern sk_fontstyleset_t sk_fontmgr_match_family (sk_fontmgr_t param0, IntPtr familyName);
```

Comment:
Could you rather edit this json file - remove the parameters that should not be String: https://github.com/mono/SkiaSharp/blob/main/binding/libSkiaSharp.json#L423-L443

Reply:
@mattleibow Thank you and I edit the json file. Could you review my code?

Reply:
@mattleibow  But I fail to re-run the generator.

```
dotnet run --project=utils/SkiaSharpGenerator/SkiaSharpGenerator.csproj -- generate --config binding/libSkiaSharp.json --skia externals/skia --output binding/Binding/SkiaApi.generated.cs
Starting C# API generation...
ERROR: skiasharp-tools: An error occurred: `One or more errors occurred. (Unable to load DLL 'libclang' or one of its dependencies
```

Reply:
I think you need to install the C++ and/or desktop workloads. Not really sure exactly - usually it fails and then install more things. And even with all that, I see it just failed again so I had to run it in the dev command prompt... Need to investigate the CppAst dependencies.

Reply:
I ran the generators locally and the code is the same, so nothing more to do.